src/phoenix/server/app.py

```diff
@@ -438,13 +452,15 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             Middleware(HeadersMiddleware),
             *prometheus_middlewares,
         ],
+        exception_handlers={HTTPException: plain_text_http_exception_handler},
         debug=debug,
     )
     app.state.read_only = read_only
     app.state.export_path = export_path
     app.include_router(v1_router)
     app.include_router(router)
     app.include_router(graphql_router)
+    app.add_middleware(GZipMiddleware)
```

Comment:
Handles compression of responses depending on whether an `Accept-Encoding` request header is present. Does not handle de-compression of compressed request content.

See https://github.com/encode/starlette/blob/6f863b0d3b8e8f18d5df9e8cd2514f7085b874e1/starlette/middleware/gzip.py#L20